### DISCUSSION TITLE
Нужно обсудить ⚠️

### DISCUSSION THREAD

**Discussion by Maks1mio** [Maks1mio/YMusic-DRPC#6] [Q&A] [answered]
Я задумываюсь о возможности внесения изменений в файл app.asar и добавления RPC в исходный код. Однако, есть опасения относительно необходимости постоянного патчинга этого файла после каждого обновления яндекс.
В таком случае можно обойтись без использования py.

### **Ваше мнение или есть какие-то предложения?**

![image](https://github.com/Maks1mio/YMusic-DRPC/assets/44835662/b47a0381-c596-44ce-9d28-8aaec5b7a64a)

**Comment by AnimeLore** [comment]
Есть ли возможность добавить что-то по типу автоматического патчера?

**Reply by Maks1mio** [reply]
На счет автоматического не знаю, просто имеет ли это смысл.

**Comment by mollyvita** [comment, marked as answer]
смысл вполне имеется, на основе патчинга app.asar работают многие моды на тот же дискорд, тем самым при правильной настройке можно сделать автообновление прямо внутри патча

**Reply by Maks1mio** [reply]
Просто у Яндекса при обновлении приложении качается новый exe и удаляет подчистую файл Яндекс Музыка в programs, затем заново распаковывает (не помню что у дискорда так)

У меня мало опыта в реализации этой штуки, я пытался, мб когда-нибудь но не сейчас

**Reply by Maks1mio** [reply]
Дискуссия https://github.com/Maks1mio/YMusic-DRPC/discussions/24
